### ISSUE TITLE
error when run preiewAuto

### ISSUE THREAD

**Issue by He-Pin** [sbt/sbt-site#119]
reproduce ,

checkout :https://github.com/ReactivePlatform/reactive-design-patterns-cn

in project `docs`, run `previewAuto`.

```
[IJ]sbt:ReactiveDesignPatterns> previewAuto
[error] java.nio.file.NoSuchFileException: /Users/kerr/IdeaProjects/reactive-design-patterns-cn/src/main/scala-2.12
[error] 	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
[error] 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
[error] 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
[error] 	at sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
[error] 	at sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:144)
[error] 	at java.nio.file.Files.readAttributes(Files.java:1737)
[error] 	at java.nio.file.FileTreeWalker.getAttributes(FileTreeWalker.java:219)
[error] 	at java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:276)
[error] 	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:322)
[error] 	at java.nio.file.FileTreeIterator.<init>(FileTreeIterator.java:72)
[error] 	at java.nio.file.Files.find(Files.java:3687)
[error] 	at com.typesafe.sbt.site.Compat$.$anonfun$genSources$3(Compat.scala:41)
[error] 	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
[error] 	at scala.collection.immutable.List.foreach(List.scala:389)
[error] 	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
[error] 	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
[error] 	at scala.collection.immutable.List.flatMap(List.scala:352)
[error] 	at com.typesafe.sbt.site.Compat$.$anonfun$genSources$2(Compat.scala:33)
[error] 	at com.typesafe.sbt.site.Preview$.apply(preview.scala:15)
[error] 	at com.typesafe.sbt.site.SitePreviewPlugin$.$anonfun$projectSettings$4(SitePreviewPlugin.scala:46)
[error] 	at com.typesafe.sbt.site.SitePreviewPlugin$.$anonfun$projectSettings$4$adapted(SitePreviewPlugin.scala:42)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:39)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:66)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:262)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:271)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:262)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:174)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:36)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (previewAuto) java.nio.file.NoSuchFileException: /Users/kerr/IdeaProjects/reactive-design-patterns-cn/src/main/scala-2.12
[error] Total time: 0 s, completed 2018-1-8 18:24:10
```

**Comment by netvl** [comment]
Same thing happens for me. Apparently, it requires the full set of directories: `src/main/{scala,scala-2.11,java,resources}`, same for `src/test`. This is especially annoying because I don't have any `src` directory in the root project (and my Paradox site resides in `paradox-site`, with directory overridden in `Compile / paradox / sourceDirectory` key).

**Comment by jonas** [comment]
Looks like a duplicate of #118. I've added a workaround and will release as part of 1.3.2.

**Comment by ennru** [comment]
@hepin1989  Was this resolved by 1.3.2? Can you close it if so, please?

**Comment by hepin1989** [comment]
I will try it later ,thanks for the ping @ennru 。
